cmake/BipedalLocomotionFrameworkFindDependencies.cmake

```diff
@@ -144,6 +144,9 @@ checkandset_dependency(Qhull)
 find_package(casadi QUIET)
 checkandset_dependency(casadi)
 
+find_package(cppad QUIET)
```

Comment:
Please add ``cppad`` as a dependency in the main CMakeLists when generating the configure file. (I know this should be automatic, but I was not able to find a way to do this cleanly).

Reply:
Actually I didn't understand this comment.

Reply:
Ok you're referring to this https://github.com/dic-iit/bipedal-locomotion-framework/blob/ea886f7060bcffea4e11fab645a7deec42a2476e/CMakeLists.txt#L83-L91

Reply:
Done in be65efda73e41d3796db8d7b1682ab314ec11f6f